src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`3.cs

```diff
@@ -139,28 +139,19 @@ protected override void SetupNewTextView(IVsTextView textView)
                     ProducerPopulatedTagSource<IOutliningRegionTag> tagSource;
                     if (outliningTaggerProvider.TryRetrieveTagSource(wpfTextView, wpfTextView.TextBuffer, out tagSource))
                     {
-                        tagSource.ComputeTagsSynchronouslyIfNoAsynchronousComputationHasCompleted = true;
-
-                        try
+                        // If this file is a metadata-from-source file, we want to force-collapse
+                        if (isOpenMetadataAsSource)
                         {
-                            // If this file is a metadata-from-source file, we want to force-collapse
-                            if (isOpenMetadataAsSource)
-                            {
-                                outliningManager.CollapseAll(new SnapshotSpan(wpfTextView.TextBuffer.CurrentSnapshot,
-                                                                              start: 0,
-                                                                              length: wpfTextView.TextBuffer.CurrentSnapshot.Length),
-                                                             c => c.Tag.IsImplementation);
-                            }
-                            else
-                            {
-                                // Set the initial outlining state by reading from the suo file, this operation requires
-                                // us to synchronously compute the outlining region tags.
-                                viewEx.PersistOutliningState();
-                            }
+                            outliningManager.CollapseAll(new SnapshotSpan(wpfTextView.TextBuffer.CurrentSnapshot,
```

Comment:
Does this call now go through the accurate tagger? It was implicitly relying on the old behavior, so hopefully it is.